ShoppingM8-be/src/main/java/edu/pw/shoppingm8/product/api/dto/ProductCreateRequestDto.java

```diff
@@ -0,0 +1,22 @@
+package edu.pw.shoppingm8.product.api.dto;
+
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.validation.constraints.NotNull;
+
+import edu.pw.shoppingm8.product.ProductCategory;
+import lombok.Value;
+
+@Value
+public class ProductCreateRequestDto {
+    @NotNull
+    String name;
+    @NotNull
+    String unit;
+    @NotNull
+    @Enumerated(EnumType.STRING)
```

Comment:
This is a database annotation. It determine whether db stores enum values as ordinals or strings

Reply:
Please, remove it

Reply:
Then which annotation to use for jsons?

Reply:
or how to specify to pass enum values with strings

Reply:
None. It works out of the box

Reply:
fixed